dvc/utils/__init__.py

```diff
@@ -232,13 +232,17 @@ def current_timestamp():
 
 
 def load_stage_file(path):
```

Comment:
*not scope of this PR* it's a bad place for a stage specific methods, we should either make them general (load_yaml_*) or move into `stage.py` or stage/something ...